autokeras/keras_layers.py

```diff
@@ -1,18 +1,13 @@
-import inspect
-
 import tensorflow as tf
 from tensorflow.keras.layers.experimental import preprocessing
-from tensorflow.python.keras.layers.preprocessing import index_lookup
 from tensorflow.python.util import nest
 
-CombinerPreprocessingLayer = inspect.getmro(preprocessing.Normalization)[1]
-Combiner = inspect.getmro(preprocessing.Normalization()._combiner.__class__)[1]
-
 INT = 'int'
 NONE = 'none'
 ONE_HOT = 'one-hot'
 
 
+@tf.keras.utils.register_keras_serializable()
 class MultiColumnCategoricalEncoding(preprocessing.PreprocessingLayer):
```

Comment:
Note that the layer is named "CategoryEncoding", so it would be more consistent to reuse the same name here, e.g. "MultiCategoryEncoding"